Wrappers/Python/ccpi/viewer/widgets/box_widgets.py

```diff
@@ -206,64 +210,72 @@ def GetBoxBoundsFromEventPosition(viewer, position, scale_factor=0.3):
 
         # Get maximum extents of the image in world coords
         world_image_max = viewer.style.GetImageWorldExtent()
-
         # Set the minimum world value
         world_image_min = (0, 0, 0)
 
         world_extent = [0, world_image_max[0], 0, world_image_max[1], 0, world_image_max[2]]
```

Comment:
Let's open an issue about this as `GetImageWorldExtent` should actually just return you the `world_extent` but it isn't.

Reply:
Opened https://github.com/TomographicImaging/CILViewer/issues/426